lib/rules/lint-no-curly-component-invocation.js

```diff
@@ -31,6 +31,7 @@ const BUILT_IN_HELPERS = [
   'unless',
   'with',
   'yield',
+  '-in-element',
```

Comment:
Lets allow both versions (hopefully some day we finally ship the public one 😸):

```suggestion
  '-in-element',
  'in-element',
```